afteraction-share.html

```diff
@@ -369,7 +369,9 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 			var winLeft = (screen.width / 2) - (popupWidth / 2);
 			window.open( destination , 'new-window-popup', 'top=' + winTop + ',left=' + winLeft + ',toolbar=0,status=0,resizable=yes,scrollbars=yes,width=' + popupWidth + ',height=' + popupHeight );
 			// send Google Analytics event
-			ga('send', 'event', 'scrolling-signup', 'share-facebook', window.location.pathname);
+			if (typeof ga !== 'undefined') {
```

Comment:
Might want to add below commentary about handling adblockers here, and might want to expand on what this is going for future devs.

What is this doing? Basically if an adblocker is present and blocks GA, this code block won't run? How is that different that what happens without this if block?

Reply:
If it runs without checking that `ga` is `undefined`, there's an exception that prevents the rest of the JS from executing correctly. I can add a comment explaining that!

Reply:
++ to adding the comment @sukhada. It will be very helpful.